appveyor.yml

```diff
@@ -0,0 +1,45 @@
+image: Visual Studio 2017
+
+# Versioning
+version: '1.0.{build}'
+
+assembly_info:
+  patch: true
+  file: 'src\Common\AzureADWindowsAuthenticator\Properties\AssemblyInfo.cs'
+  assembly_version: '{version}'
+  assembly_file_version: '{version}'
+  assembly_informational_version: '{version}'
+
+dotnet_csproj:
+  patch: true
+  file: 'src\Directory.Build.props'
+  version: '{version}'
+  package_version: '{version}'
+  assembly_version: '{version}'
+  file_version: '{version}'
+  informational_version: '{version}'
+
+# Install
+install:
+  - ps: 'scripts\DisableStrongName.ps1'
```

Comment:
It might be helpful to add comments here or in the script itself stating why this is necessary.

Reply:
I will add it to the script